Exception Handling/Task1/Program.cs

```diff
@@ -1,10 +1,34 @@
-﻿namespace Task1
+﻿using System;
+
+namespace Task1
 {
     internal class Program
     {
         private static void Main(string[] args)
         {
             // TODO: Implement the task here.
```

Comment:
@IhorStetskyi, please remove comments that have already been fixed.

Reply:
removed old comments